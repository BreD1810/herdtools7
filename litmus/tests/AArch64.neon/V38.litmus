AArch64 V38
(* Test EOR (vector) *)
{ 0:X0=x; 0:X1=y; }

P0;
  MOVI V0.8B,#1             ;
  MOVI V1.8B,#2             ;
  EOR V2.8B, V0.8B, V1.8B   ;
  ST1 {V2.B}[0],[X0]        ;
  ST1 {V2.B}[9],[X1]        ;

forall (x=3 /\ y=0)
