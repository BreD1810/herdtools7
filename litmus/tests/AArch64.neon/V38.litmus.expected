Test V38 Required
Histogram (1 states)
1000000:>x=3; y=0;
Ok

Witnesses
Positive: 1000000, Negative: 0
Condition forall (x=3 /\ y=0) is validated
Hash=c4eb299546fb85c0a9c558a1637a28c0
Observation V38 Always 1000000 0
